src/helpers/auth/index.js

```diff
@@ -0,0 +1,233 @@
+/**
```

Comment:
+1 to this file. However, I'm curious if by convention these should be PascalCase like they are?

PascalCase is generally reserve for Components. While this are function which take a Component and return a Component. For instance `isolate()` is not a capitalized function and it shares the same signature

```haskell
utility :: Component -> Component
```